dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tag.java

```diff
@@ -44,6 +45,11 @@
 @SuppressWarnings("checkstyle:magicnumber")
 public class Tag extends Version<Tag> implements Comparable<Tag> {
 
+    @Column
+    @JsonProperty("last_build")
+    @ApiModelProperty(value = "The last time this tag was built gets stored here.", position = 19)
```

Comment:
1. I would drop `gets stored here`.
2. I think this should more clearly state that it's the Docker image that was built. Not sure how to word it though :)

Reply:
I think maybe the word container would work for your second point. I know it is also used by Docker, but I still think container is more general of a concept.

Maybe "The last time the container was built"

Reply:
"The last time the container backing this tool version was built" maybe?